### PR TITLE
Fix simba jdbc tests that use session variables

### PR DESCRIPTION
**Pull request by brian-rickman**
1st commit (d0dee44) has already been reviewed and merged to sprint-48.  

Only the second commit (3908ed4) needs to be reviewed now.

### CODE REVIEW

File: presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java [hunk 1]
```diff
@@ -45,6 +46,28 @@ private static String getSessionProperty(Connection connection, String key, Stri
...
+        // Everything else should be enclosed in single quotes.
+
+        if (value.equalsIgnoreCase("true") ||
+                value.equalsIgnoreCase("false")) {
+            return false;
+        }
+
+        // Not particularly efficient, but for occasional use in
+        // setting session properties for tests, this should be OK.
+        try {
```
Comment (amrutagokhale) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java:74]: Can you use `StringUtils. isNumeric()` first, to check if the value is a number other than a decimal?

File: presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java [hunk 2]
```diff
@@ -45,6 +46,28 @@ private static String getSessionProperty(Connection connection, String key, Stri
...
+            new BigDecimal(value);
+            return false;
+        }
+        catch (NumberFormatException e) { }
```
Comment (amrutagokhale) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java]: may be, print the error message?

**Review by amrutagokhale — APPROVED** [review]
A couple of suggestions. Looks good.

File: presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java (hunk 1 above)
Comment (brian-rickman) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java:74]: Good idea.  I'll add that.

File: presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java (hunk 2 above)
Comment (brian-rickman) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java]: Sure.  I'll log it as Info.

**Comment by amrutagokhale** [comment]
Changes look good.

**Comment by brian-rickman** [comment]
This PR is an extension of #486

**Pull request closed without merging**